.github/workflows/CI.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:  
+  pull_request:    
```

Comment:
Just a question, what happens if you remove the `on` section?

If it results in the pipelines running always (pull requests included), I'm all in favor! I'd like pipelines to run in the master branch as well when changes are integrated. :blush:

Reply:
Names is needed to identify the action, but perhaps '''on_push''' is what is needed instead of '''on_pullrequest''' that we have currently

Reply:
We'll leave it as-is then. We can change/update it in the future! :blush: